<!DOCTYPE html>
<html lang="en-US">
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">

<!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Awesome Cytodata | Awesome Lists with GitHub stars</title>
<meta name="generator" content="Jekyll v3.9.5">
<meta property="og:title" content="Awesome Cytodata">
<meta property="og:locale" content="en_US">
<link rel="canonical" href="https://awesome.imyz.dev/lists/cytodata-awesome-cytodata.html">
<meta property="og:url" content="https://awesome.imyz.dev/lists/cytodata-awesome-cytodata.html">
<meta property="og:site_name" content="Awesome Lists with GitHub stars">
<meta property="og:type" content="website">
<meta name="twitter:card" content="summary">
<meta property="twitter:title" content="Awesome Cytodata">
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","headline":"Awesome Cytodata","url":"https://awesome.imyz.dev/lists/cytodata-awesome-cytodata.html"}</script>
<!-- End Jekyll SEO tag -->

    <link rel="stylesheet" href="/assets/css/style.css?v=d7674fd6c2505d33010aa0eb465ff1f3bdd5d931">
    <!-- start custom head snippets, customize with your own _includes/head-custom.html file -->

<!-- Setup Google Analytics -->



<!-- You can set your favicon here -->
<!-- link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" -->

<!-- end custom head snippets -->

  </head>
  <body>
    <div class="container-lg px-3 my-5 markdown-body">
      
      <h1><a href="https://awesome.imyz.dev/">Awesome Lists with GitHub stars</a></h1>
      

      <h1 id="awesome-cytodata-">Awesome Cytodata <a href="https://awesome.re"><img src="https://awesome.re/badge.svg" alt="Awesome" referrerpolicy="no-referrer" loading="lazy"></a>
</h1>

<blockquote>
  <p>A curated list of awesome cytodata resources.</p>
</blockquote>

<p><img src="https://github.com/cytodata/awesome-cytodata/raw/master/cytodata-logo.png" alt="cytodata logo" referrerpolicy="no-referrer" loading="lazy"></p>

<p><a href="https://cytodata.org/">Cytodata</a> refers to a community of researchers and resources involved in the <strong>image-based profiling</strong> of <strong>biological phenotypes</strong>.
These <strong>biological phenotypes</strong> are typically induced by genetic or chemical perturbations and often represent disease states.
<strong>Image-based profiling</strong> is used to inspect these phenotypes to uncover biological insight including discovering the impact of genetic alterations and determining the mechanism of action of compounds.</p>

<p>This page represents a curated list of software, datasets, landmark publications, and image-based profiling methods.
Our goal is to provide researchers, both new and established, a place to discover and document awesome Cytodata resources.</p>

<h2 id="contents">Contents</h2>

<ul>
  <li>
<a href="#datasets">Datasets</a>
    <ul>
      <li><a href="#raw-images">Raw Images</a></li>
      <li><a href="#chemical-perturbations">Chemical Perturbations</a></li>
      <li><a href="#genetic-perturbations">Genetic Perturbations</a></li>
    </ul>
  </li>
  <li><a href="#software">Software</a></li>
  <li>
<a href="#publications">Publications</a>
    <ul>
      <li><a href="#reviews">Reviews</a></li>
      <li><a href="#applications">Applications</a></li>
      <li><a href="#methods">Methods</a></li>
    </ul>
  </li>
</ul>

<h2 id="datasets">Datasets</h2>

<p>Annotated datasets, including <strong>raw images</strong> and <strong>processed profiles</strong>, for image-based profiling of chemical and genetic perturbations.</p>

<h3 id="raw-images">Raw Images</h3>
<ul>
  <li>
<a href="https://broad.io/CellPaintingGallery">The Cell Painting Gallery</a> - The Cell Painting Gallery is a collection of image datasets created using the Cell Painting assay (or similar); it is maintained by the Carpenter–Singh lab at the Broad Institute.</li>
  <li>
<a href="https://data.broadinstitute.org/bbbc/">Broad Bioimage Benchmark Collection</a> - The Broad Bioimage Benchmark Collection (BBBC) is a collection of freely downloadable microscopy image sets. In addition to the images themselves, each set includes a description of the biological application and some type of “ground truth” (expected results).</li>
  <li>
<a href="https://idr.openmicroscopy.org/">Image Data Resource</a> - Public repository of image datasets from published scientific studies.</li>
  <li>
<a href="https://www.rxrx.ai/rxrx1">RxRx1</a> - RxRx1 is a set of 125,514 high-resolution 512x512 6-channel fluorescence microscopy images of human cells under 1,108 genetic perturbations in 51 experimental batches across four cell types.  The images were produced by Recursion Pharmaceuticals in their labs in Salt Lake City, Utah.  Researchers will use this dataset for studying and benchmarking methods for dealing with biological batch effects, as well as areas in machine learning such as domain adaptation, transfer learning, and k-shot learning.</li>
  <li>
<a href="https://www.rxrx.ai/rxrx19">RxRx19</a> - RxRx19 is the first morphological dataset that demonstrates the rescue of morphological effects of COVID-19.</li>
  <li>
<a href="https://www.proteinatlas.org/humanproteome/subcellular">Human Protein Atlas</a> - Among other assays, the HPA performed confocal imaging of displaying the location of more than 2/3 of human proteins in cell lines. <a href="https://github.com/CellProfiling/HPA-competition#script-to-download-hpav18">Raw images <img src="https://img.shields.io/github/stars/CellProfiling/HPA-competition" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/CellProfiling/HPA-competition" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a> or <a href="https://www.proteinatlas.org/about/download">infered protein subcellular locations</a> can be downloaded.</li>
</ul>

<h3 id="chemical-perturbations">Chemical Perturbations</h3>

<ul>
  <li>
<a href="https://doi.org/10.1371/journal.pone.0080999">Gustafsdottir et al. 2013</a> - Cell painting profiles from 1,600 bioactive compounds in U2OS cells (Access from public S3 bucket: <code class="language-plaintext highlighter-rouge">s3://cytodata/datasets/Bioactives-BBBC022-Gustafsdottir/profiles/Bioactives-BBBC022-Gustafsdottir/</code>).</li>
  <li>
<a href="https://doi.org/10.1073/pnas.1410933111">Wawer et al. 2014</a> - Cell painting profiles from 31,770 compounds in U2OS cells (<a href="http://www.broadinstitute.org/mlpcn/data/Broad.PNAS2014.ProfilingData.zip">Click to download</a>).</li>
  <li>
    <table>
      <tbody>
        <tr>
          <td>
<a href="https://doi.org/10.1093/gigascience/giw014">Bray et al. 2017</a> - Cell painting profiles from 30,616 compounds in U2OS cells (Center Driven Research Project <em>CDRP</em>) (<a href="https://doi.org/10.5524/100351">Download from GigaDB</a>
</td>
          <td>Access from public S3 bucket: <code class="language-plaintext highlighter-rouge">s3://cytodata/datasets/CDRPBIO-BBBC036-Bray/profiles_cp/CDRPBIO-BBBC036-Bray/</code>).</td>
        </tr>
      </tbody>
    </table>
  </li>
  <li>
<a href="https://doi.org/10.1038/s41592-022-01667-0">Haghighi et al. 2021</a> - Cell painting matched to L1000 profiles in 4 experiments, including compound and genetic screens (<a href="https://github.com/carpenterlab/2021_Haghighi_submitted">Details on GitHub <img src="https://img.shields.io/github/stars/carpenterlab/2021_Haghighi_submitted" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/carpenterlab/2021_Haghighi_submitted" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a>).</li>
</ul>

<h3 id="genetic-perturbations">Genetic Perturbations</h3>

<ul>
  <li>
<a href="https://doi.org/10.1371/journal.pone.0131370">Singh et al. 2015</a> - 3,072 cell painting profiles from 41 genes knocked down with RNA interference (RNAi) in U2OS cells (<a href="https://github.com/carpenterlab/2016_bray_natprot/blob/6dcdcf72cd90bb2dbf238b3ecf94691246d8f104/supplementary_files/profiles.csv.zip">Access from GitHub <img src="https://img.shields.io/github/stars/carpenterlab/2016_bray_natprot" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/carpenterlab/2016_bray_natprot" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a>).</li>
  <li>
<a href="https://doi.org/10.7554/eLife.24060.001">Rohban et al. 2017</a> - Cell painting data from 220 overexpressed genes in U2OS cells (Access from public S3 bucket: <code class="language-plaintext highlighter-rouge">s3://cytodata/datasets/TA-ORF-BBBC037-Rohban/profiles_cp/TA-ORF-BBBC037-Rohban/</code>).</li>
  <li>Unpublished - Cell painting profiles of 596 overexpressed alleles from 53 genes in A549 cells (Access from public S3 bucket: <code class="language-plaintext highlighter-rouge">s3://cytodata/datasets/LUAD-BBBC043-Caicedo/profiles_cp/LUAD-BBBC043-Caicedo/</code>)</li>
  <li>Unpublished - 3,456 cell painting profiles from CRISPR experiments knocking down 59 genes in A549, ES2, and HCC44 cells (<a href="https://github.com/broadinstitute/cell-health/tree/master/0.generate-profiles/data/profiles">Access from GitHub <img src="https://img.shields.io/github/stars/broadinstitute/cell-health" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/broadinstitute/cell-health" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a>).</li>
</ul>

<h2 id="software">Software</h2>

<p>Open source software packages for image-based profiling of biological phenotypes.</p>

<ul>
  <li>
<a href="https://www.cellclassifier.org/">Advanced Cell Classifier</a> - A software package for exploration, annotation and classification of cells within large datasets using machine learning.</li>
  <li>
<a href="http://cellprofiler.org/">CellProfiler</a> - CellProfiler is a free open-source software for measuring and analyzing cell images.</li>
  <li>
<a href="http://cellprofiler.org/cp-analyst/">CellProfiler Analyst</a> - Interactive data exploration, analysis, and classification of large biological image sets.</li>
  <li>
<a href="https://github.com/cytomining/cytominer">Cytominer <img src="https://img.shields.io/github/stars/cytomining/cytominer" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/cytomining/cytominer" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a> - Methods for image-based cell profiling in R.</li>
  <li>
<a href="https://github.com/aoles/EBImage">EBImage <img src="https://img.shields.io/github/stars/aoles/EBImage" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/aoles/EBImage" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a> - Image processing toolbox for R.</li>
  <li>
<a href="http://htsvis.dkfz.de/HTSvis/">HTSvis</a> - A web app for exploratory data analysis and visualization of arrayed high-throughput screens.</li>
  <li>
<a href="https://github.com/menchelab/BioProfiling.jl">BioProfiling.jl <img src="https://img.shields.io/github/stars/menchelab/BioProfiling.jl" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/menchelab/BioProfiling.jl" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a> - Toolkit for filtering and curation of morphological profiles in Julia.</li>
  <li>
<a href="https://github.com/cytomining/pycytominer">PyCytominer <img src="https://img.shields.io/github/stars/cytomining/pycytominer" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/cytomining/pycytominer" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a> - Methods for image-based cell profiling in Python.</li>
  <li>
<a href="https://imjoy.io">ImJoy</a> - A platform compiling tool for deep-learning based image analyses with a GUI.</li>
  <li>
<a href="https://github.com/BodenmillerGroup/histoCAT">histoCAT <img src="https://img.shields.io/github/stars/BodenmillerGroup/histoCAT" alt="GitHub Repo Stars" referrerpolicy="no-referrer" loading="lazy"> <img src="https://img.shields.io/github/last-commit/BodenmillerGroup/histoCAT" alt="GitHub last commit" referrerpolicy="no-referrer" loading="lazy"></a> - Toolbox to extract quantitative phenotypic descriptors and contextual information for histology and multiplex imaging.</li>
</ul>

<h2 id="publications">Publications</h2>

<p>Publications related to image-based profiling.</p>

<h3 id="reviews">Reviews</h3>

<ul>
  <li>
<a href="https://www.nature.com/articles/s41573-020-00117-w">Image-based profiling for drug discovery: due for a machine-learning upgrade?</a> - 2020 review of applications in image-based profiling from a Carpenter lab/pharma perspective.</li>
  <li>
<a href="https://doi.org/10.1038/nmeth.4397">Data-analysis strategies for image-based cell profiling</a> - Introduce the steps required to create high-quality image-based (i.e., morphological) profiles from a collection of microscopy images.</li>
  <li>
<a href="https://doi.org/10.1016/j.tcb.2016.03.008">High-content screening for quantitative cell biology</a> - Describe some recent applications of HCS, ranging from the identification of genes required for specific biological processes to the characterization of genetic interactions.</li>
  <li>
<a href="https://doi.org/10.1016/j.cell.2015.11.007">Microscopy-based high-content screening</a> - Describe the state of the art for image-based screening experiments and delineate experimental approaches and image-analysis approaches as well as discussing challenges and future directions, including leveraging CRISPR/Cas9-mediated genome engineering.</li>
  <li>
<a href="https://doi.org/10.1016/j.copbio.2016.04.003">Applications in image-based profiling of perturbations</a> - Describe applications of image-based profiling including target and MOA identification, lead hopping, library enrichment, gene annotation and identification of disease-specific phenotypes</li>
  <li>
<a href="https://doi.org/10.1002/cyto.a.22909">Large-scale image-based screening and profiling of cellular phenotypes</a> - Overview of image-based profiling, including its applications and challenges.</li>
  <li>
<a href="https://dx.doi.org/10.1002%2Fbies.201400011">How cells explore shape space: A quantitative statistical perspective of cellular morphogenesis</a> - Discussion on the biology of cell shape changes based on quantitative descriptors.</li>
  <li>
<a href="https://doi.org/10.1016/J.COISB.2018.05.004">Machine learning and image-based profiling in drug discovery</a> - Introduction to morphological profiling and discussion on what machine learning has to offer.</li>
  <li>
<a href="https://doi.org/10.15252/msb.202110768">Pooled genetic screens with image-based profiling</a> - Review of the different modalities available for genetic screens and which ones are suitable for morphological profiling.</li>
</ul>

<h3 id="collections">Collections</h3>

<ul>
  <li>
<a href="https://www.nature.com/collections/cfcdjceech">Deep learning in microscopy</a> - A collection of review and research articles published in Nature Methods related to multiple use cases of deep learning, including noise reduction, segmentation, tracking and representation learning.</li>
  <li>
<a href="https://journals.sagepub.com/toc/jbxb/25/7">High-Content Imaging and Informatics</a> - A collection of high-content imaging method and application articles published in SLAS Discovery.</li>
</ul>

<h3 id="applications">Applications</h3>

<ul>
  <li>
<a href="https://rdcu.be/ccBFH">Expanding the antibacterial selectivity of polyether ionophore antibiotics through diversity-focused semisynthesis</a> - Poulsen lab paper from 2020 where antibiotics are tested for their ability to leave mammalian cells as intact as possible, per the Cell Painting assay.</li>
  <li>
<a href="https://doi.org/10.1038/nmeth1032">Image-based multivariate profiling of drug responses from single cells</a> - A multivariate method for classifying untreated and treated human cancer cells based on ∼300 single-cell phenotypic measurements.</li>
  <li>
<a href="https://doi.org/10.1016/j.molmet.2019.03.001">Discovering metabolic disease gene interactions by correlated effects on cellular morphology</a> - Profiling disease-gene interaction during adipocyte differentiation.</li>
  <li>
<a href="https://doi.org/10.1038/nature08869">Phenotypic profiling of the human genome by time-lapse microscopy reveals cell division genes</a> - This study provides an in-depth analysis of cell division phenotypes and makes the entire high-content data set available as a resource to the community.</li>
  <li>
<a href="https://doi.org/10.1016/j.taap.2019.114876">Bioactivity screening of environmental chemicals using imaging-based high-throughput phenotypic profiling</a> - Use of image-based profiling to screen the bioactivity of environmental chemicals</li>
  <li>
<a href="https://doi.org/10.1016/j.chembiol.2018.01.015">Repurposing High-Throughput Image Assays Enables Biological Activity Prediction for Drug Discovery</a> - Using image-based profiles to predict the bioactivity of small molecules in other unrelated assays.</li>
  <li>
<a href="https://doi.org/10.1038/s41598-020-69354-8">Tales of 1,008 Small Molecules: Phenomic Profiling through Live-cell Imaging in a Panel of Reporter Cell Lines</a> - Demonstrating the effects of polypharmacology in MOA prediction while offering solutions for overcoming it in future image-based profiling studies.</li>
  <li>
<a href="https://doi.org/10.1038/s41467-019-13058-9">Mapping the perturbome network of cellular perturbations</a> - Image-based profiling and network analysis of drug combinations.</li>
  <li>
<a href="https://doi.org/10.1016/j.celrep.2021.109318">Morphological profiling of human T and NK lymphocytes by high-content cell imaging</a> - Image-based profiling of actin organization at the immunological synapse.</li>
  <li>
<a href="https://doi.org/10.1126/science.aal3321">A subcellular map of the human proteome</a> - Classification of protein subcellular location from confocal microscopy images of the Human Protein Atlas.</li>
  <li>
<a href="https://doi.org/10.1038/s41586-021-04115-9">A multi-scale map of cell structure fusing protein images and interactions</a> - Combining confocal imaging and mass spectrometry representations of proteins to predict physical proximity and characterize cellular organization.</li>
  <li>
<a href="https://doi.org/10.1091/mbc.E20-12-0784">Predicting cell health phenotypes using image-based morphology profiling</a> - Image-based profiles as predictors of apoptosis, proliferation and other cell health descriptors.</li>
  <li>
<a href="https://doi.org/10.15252/msb.20199243">Systematic genetics and single‐cell imaging reveal widespread morphological pleiotropy and cell‐to‐cell variability</a> - Analysis of single-cell profiles to characterize variability, pleiotropy and incomplete penetrance.</li>
  <li>
<a href="https://doi.org/10.15252/msb.20178064">Large‐scale image‐based profiling of single‐cell phenotypes in arrayed CRISPR‐Cas9 gene perturbation screens</a> - Demonstrates feasibility of imaging arrayed CRISPR screens and offers a way to characterize transfection efficacy in individual cells.</li>
  <li>
<a href="https://doi.org/10.1038/s41467-022-30722-9">Multiparametric phenotyping of compound effects on patient derived organoids</a> - Profiling chemical effects on patient-derived organoids.</li>
  <li>
<a href="https://doi.org/10.15252/MSB.20156400">A chemical-genetic interaction map of small molecules using high-throughput imaging in cancer cells</a> - Profiling the morphological changes induced in 1280 compounds in 12 knockout cell lines.</li>
  <li>
<a href="https://doi.org/10.7554/eLife.40174">Time-resolved mapping of genetic interactions to model rewiring of signaling pathways</a> - Changes in genetic interactions across time based on multiple morphological descriptors.</li>
  <li>
<a href="https://doi.org/10.1016/j.cels.2019.09.002">High-Content Imaging of Unbiased Chemical Perturbations Reveals that the Phenotypic Plasticity of the Actin Cytoskeleton Is Constrained</a> - Defining morphological clusters in a large compound screen.</li>
  <li>
<a href="https://doi.org/10.7554/eLife.05464">A map of directional genetic interactions in a metazoan cell</a> - Characterizing genetic interactions by integrating 21 phenotypic descriptors.</li>
  <li>
<a href="https://doi.org/10.1016/j.cell.2022.10.017">The phenotypic landscape of essential human genes</a> - Comparing morphological descriptors in a pooled CRISPR screen with in-situ sequencing</li>
  <li>
<a href="https://doi.org/10.1101/580654">Evaluation of Gene Expression and Phenotypic Profiling Data as Quantitative Descriptors for Predicting Drug Targets and Mechanisms of Action</a> - Benchmarking profiling modalities, including image-based profiles, for mechanism of action prediction.</li>
  <li>
<a href="https://doi.org/10.15252/msb.202211087">The molecular architecture of cell cycle arrest</a> - Comparing cellular features across stages of the cell cycle.</li>
  <li>
<a href="https://doi.org/10.1038/s41586-022-05563-7">Integrated intracellular organization and its variations in human iPS cells</a> - Decomposing cellular and nuclear shapes in 3D in multiple iPSC and studying association between cellular structures.</li>
  <li>
<a href="https://doi.org/10.1038/s41587-020-0651-8">Single-cell metabolic profiling of human cytotoxic T cells</a> - Combining metabolic profiling and spatial information to define immune subsets in tumor microenvironments.</li>
  <li>
<a href="https://doi.org/10.1038/s41586-019-1876-x">The single-cell pathology landscape of breast cancer</a> - Defining cell populations and their interactions in breast cancer based on shape, intensity and contextual information from multiplexed imaging.</li>
  <li>
<a href="https://doi.org/10.1101%2Fgr.276059.121">Identification of phenotype-specific networks from paired gene expression–cell shape imaging data</a> - Looking for gene networks underlying cellular morphology by matching expression and imaging data.</li>
  <li>
<a href="https://doi.org/10.1371/journal.pcbi.1009888">Predicting drug polypharmacology from cell morphology readouts using variational autoencoder latent space arithmetic</a> - Model cell morphology with autoencoders to estimate effects of drug combinations.</li>
  <li>
<a href="https://doi.org/10.1158/2643-3230.BCD-21-0219">Deep Morphology Learning Enhances Ex Vivo Drug Profiling-Based Precision Medicine</a> - Concrete description of how morphological information can be extracted from patient material and guide treatment.</li>
</ul>

<h3 id="methods">Methods</h3>

<ul>
  <li>
<a href="https://doi.org/10.1038/nprot.2016.105">Cell Painting, a high-content image-based assay for morphological profiling using multiplexed fluorescent dyes</a> - Protocol describing the design and execution of experiments using Cell Painting.</li>
  <li>
<a href="https://doi.org/10.1371/journal.pone.0080999">Multiplex Cytological Profiling Assay to Measure Diverse Cellular States</a> - Cell Painting assay.</li>
  <li>
<a href="https://doi.org/10.1038/nmeth.3323">CIDRE: an illumination-correction method for optical microscopy</a> - Retrospective method for illumination-correction based on energy minimization.</li>
  <li>
<a href="https://doi.org/10.1046/j.1365-2818.2000.00669.x">Retrospective shading correction based entropy minimization</a> - Method for retrospective shading correction based on entropy minimization.</li>
  <li>
<a href="https://doi.org/10.1038/s41467-019-10154-8">Capturing single-cell heterogeneity via data fusion improves image-based profiling</a> - Adds dispersion and covariances to population averages to capture single-cell heterogeneity.</li>
  <li>
<a href="https://doi.org/10.1142/S0219720005001004">Minimum redundancy feature selection from microarray gene expression data</a> - Minimum redundancy - maximum relevance feature selection framework.</li>
  <li>
<a href="https://doi.org/10.1371/journal.pcbi.1007348">Learning unsupervised feature representations for single cell microscopy images with paired cell inpainting</a> - Selfsupervised method to learn feature representations of single cells in microscopy images without labelled training data.</li>
  <li>
<a href="https://doi.org/10.1109/CVPR.2018.00970">Weakly supervised learning of single-cell feature embeddings</a> - Training CNNs using a weakly supervised approach for feature learning.</li>
  <li>
<a href="https://doi.org/10.1021/acs.jcim.8b00670">Accurate Prediction of Biological Assays with High-Throughput Microscopy Images and Convolutional Networks</a> - End-to-end learning with CNNs to predict bioactivity of small molecules in unrelated assays using image-based profiles.</li>
  <li>
<a href="https://doi.org/10.1002/cyto.a.23863">Evaluation of Deep Learning Strategies for Nucleus Segmentation in Fluorescence Images</a> - Comparing several deep learning methods for nuclear segmentation.</li>
  <li>
<a href="https://doi.org/10.1101/085118">Automating Morphological Profiling with Generic Deep Convolutional Networks</a> - Transfer of activation features of generic CNNs to extract features for image-based profiling.</li>
  <li>
<a href="https://doi.org/10.1038/ncomms14836">A BaSiC tool for background and shading correction of optical microscopy images</a> - Illumination-correction method accounting for space- and time-dependent biases.</li>
  <li>
<a href="https://doi.org/10.1038/s41592-020-01018-x">Cellpose: a generalist algorithm for cellular segmentation</a> - Generalist deep learning model for cell and nucleus segmentation with pre-trained weights.</li>
  <li>
<a href="https://doi.org/10.1371/journal.pcbi.1005177">Deep Learning Automates the Quantitative Analysis of Individual Cells in Live-Cell Imaging Experiments</a> - DeepCell: collection of deep learning segmentation models.</li>
  <li>
<a href="https://doi.org/10.1101/161422">Improving Phenotypic Measurements in
High-Content Imaging Screens</a> - Embedding single-cell and compound profiles using transfer learning, examplified on mechanism of action prediction.</li>
  <li>
<a href="https://doi.org/10.1177/1087057112469257">The Multidimensional Perturbation Value</a> - Proposing a score to define significant activity in screens.</li>
  <li>
<a href="https://doi.org/10.1038/s41598-022-12914-x">Label-Free Prediction of Cell Painting from Brightfield Images</a> - Reconstructing images for Cell Painting dyes and ensuring corresponding morphological measurements are preserved.</li>
  <li>
<a href="https://doi.org/10.3389/fbinf.2022.788607">ShapoGraphy: A User-Friendly Web Application for Creating Bespoke and Intuitive Visualisation of Biomedical Data</a> - Method to visualize morphological profiles.</li>
  <li>
<a href="https://doi.org/10.1101/227645">CytoGAN: Generative Modeling of Cell Images</a> - Generative network displaying potential for learning latent representation of biological conditions from cell images.</li>
  <li>
<a href="https://doi.org/https://doi.org/10.1093/bioinformatics/btaa905">Self-supervised feature extraction from image time series in plant phenotyping using triplet networks</a> - Direct extraction of phenotypic features from plant images.</li>
  <li>
<a href="https://doi.org/10.1016/j.cels.2022.10.001">Morphology and gene expression profiling provide complementary information for mapping cell state</a> - Comparison of the information contained in Cell Painting and L1000 assays for the same perturbations.</li>
  <li>
<a href="https://doi.org/10.1093/bioinformatics/btab497">Fully unsupervised deep mode of action learning for phenotyping high-content cellular images</a> - Unsupervised approach to represent cellular morphology with clusters corresponding to meaningful relations such as mechanism of action. With an overview of deep learning methods for morphological profiling and classification.</li>
  <li>
<a href="https://doi.org/10.1038/s41592-022-01508-0">Automated high-speed 3D imaging of organoid cultures with multi-scale phenotypic quantification</a> - Experimental and computational workflow to extract 3D morphological descriptors of organoids using light-sheet microscopy.</li>
</ul>

<h2 id="contribute">Contribute</h2>

<p>Contributions welcome! Read the <a href="contributing.md">contribution guidelines</a> first.</p>

<h2 id="license">License</h2>

<p><a href="http://creativecommons.org/publicdomain/zero/1.0"><img src="http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg" alt="CC0" referrerpolicy="no-referrer" loading="lazy"></a></p>


      
    </div>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/4.1.0/anchor.min.js" integrity="sha256-lZaRhKri35AyJSypXXs4o6OPFTbTmUoltBbDCbdzegg=" crossorigin="anonymous"></script>
    <script>anchors.add();</script>
  </body>
</html>
